CHANGELOG.md

```diff
@@ -1,204 +1,256 @@
 # Change Log
 
-## [v0.1.1](https://github.com/skippbox/kompose/tree/v0.1.1) (2016-10-06)
-[Full Changelog](https://github.com/skippbox/kompose/compare/v0.1.0...v0.1.1)
+## [v0.1.2](https://github.com/kubernetes-incubator/kompose/tree/v0.1.2) (2016-10-31)
+[Full Changelog](https://github.com/kubernetes-incubator/kompose/compare/v0.1.1...v0.1.2)
 
 **Implemented enhancements:**
```

Comment:
I generated CHANGELOG using [this tool](https://github.com/skywinder/github-changelog-generator). Issues marked with `enhancement` label will be sorted into implemented enhancements, `bug` label will be fixed bugs, otherwise are closed issues 😃